src/Http/Headers/src/EntityTagHeaderValue.cs

```diff
@@ -151,36 +152,36 @@ public bool Compare(EntityTagHeaderValue other, bool useStrongComparison)
         public static EntityTagHeaderValue Parse(StringSegment input)
         {
             var index = 0;
-            return SingleValueParser.ParseValue(input, ref index);
+            return SingleValueParser.ParseValue(input, ref index)!;
         }
 
         public static bool TryParse(StringSegment input, out EntityTagHeaderValue parsedValue)
         {
             var index = 0;
-            return SingleValueParser.TryParseValue(input, ref index, out parsedValue);
+            return SingleValueParser.TryParseValue(input, ref index, out parsedValue!);
         }
 
-        public static IList<EntityTagHeaderValue> ParseList(IList<string> inputs)
+        public static IList<EntityTagHeaderValue> ParseList(IList<string>? inputs)
```

Comment:
ParseList.ParseList doesn't allow a null list be EntityTagHeaderValue does. Intended?

Reply:
Did you mean TryParseList? I updated those to also allow nulls since the code that this calls is null tolerant.